compiler/lib/config.mli

```diff
@@ -104,3 +105,13 @@ module Param : sig
 
   val lambda_lifting_baseline : unit -> int
 end
+
+(****)
+
+(** {2 Parameters that are constant across a program run} *)
+
+(** These parameters should be set at most once at the beginning of the program. *)
```

Comment:
Maybe we should make `target` fail if `set_target` was not called first. That way we can spot uninitialized codepath.

Reply:
Good idea, done.